examples/doubleclick-rtc.amp.html

```diff
@@ -30,5 +30,17 @@ <h2>Doubleclick with RTC</h2>
             rtc-config='{"urls": ["https://www.example.biz/?pvi=PAGEVIEWID&href=HREF&ds=ATTR(data-slot)&h=ATTR(height)&w=ATTR(width)&ovw=ATTR(data-override-width)&json=ATTR(data-json)&ovh=ATTR(data-override-height)"], "timeoutMillis": 500}'
             json='{"targeting":{"food":["burgers","chicken"]},"categoryExclusions":["health"],"tagForChildDirectedTreatment":0}'>
     </amp-ad>
+
+    <amp-ad width="320" height="50"
+            type="doubleclick"
+            data-slot="/4119129/mobile_ad_banner"
+            rtc-config='{
```

Comment:
Nit: could just make all rtc-config be on one line, like on line 30 above. Or could indent the JSON object (e.g. 2 more spaces on line 38, 4 more on line 39, etc. Or not, up to you.

Reply:
Going to leave as is for my own readability. And I think I am the only one who ever looks at this file anyways :D